listener/security/security.js

```diff
@@ -85,8 +90,7 @@ async function handleTooManyAttempts(requestKey, requestObject, user) {
     }
     // If 5 failed attempts have been made, lock the user out for 5 minutes
     else if (user.Attempt === 5) {
```

Comment:
For safety, if somehow we reached a state where we skipped 5.

```suggestion
    else if (user.Attempt >= 5) {
```